src/chart/components/y_axis/y-axis.component.ts

```diff
@@ -136,6 +139,14 @@ export class YAxisComponent extends ChartBaseElement {
 				() => this.state.type === 'percent' && this.scale.haltAnimation(),
 			),
 		);
+
+		this.addRxSubscription(
+			merge(this.scale.yChanged, this.chartResizeHandler.canvasResized)
+				.pipe(throttleTime(50, animationFrameScheduler, { trailing: true, leading: true }))
+				.subscribe(() => {
+					this.updateOrderedLabels();
+				}),
+		);
```

Comment:
You don't need to add new effect. Take a look at `y-axis-labels.model.ts`. It already contains effect for updating ordered labels, you should add `this.chartResizeHandler.canvasResized`, and it should work as you want